modules/installation/pages/proxy-setup.adoc

```diff
@@ -194,46 +194,72 @@ filename: "pxelinux.0"
+
 Shut down the old proxy, and leave it installed while you prepare the replacement.
 Create a reactivation key for this system and then register the new proxy using the reactivation key.
 If you do not use the reactivation key, you will need to re-register all the clients against the new proxy.
 
-[NOTE]
-====
-The reactivation key is only needed if you do not want to lose the history of the machine.
-If you do not use a reactivation key, the replacement proxy will become a "new" one with a new ID.
-====
 
 
-[[proxy.migration3.replace]]
 .Procedure: Replacing a Proxy and Keeping the Clients Registered
-. Before starting the actual migration procedure, save the data from the old proxy, if needed.
-    Consider copying important or manually created data to a central place that can also be accessed by the new proxy.
+. Before starting the migration, save the data from the old proxy, if needed.
+  Consider copying important or manually created data to a central place that can also be accessed by the new proxy.
 . Shut down the proxy.
 . Install a new {productname} Proxy.
-    For installation instructions, see xref:install-proxy-unified.adoc[Proxy Installation].
+  For installation instructions, see xref:install-proxy-unified.adoc[Proxy Installation].
 . In the {productname} {webui}, select the newly installed {productname} Proxy, and delete it from the systems list.
 . In the {webui}, create a reactivation key for the old proxy system: On the System Details tab of the old proxy click [guimenu]``Reactivation``.
   Click [guimenu]``Generate New Key``, and make a note of the new key.
 . Register the proxy with a bootstrap script as described in xref:installation:proxy-registration.adoc[].
-  In the bootstrap script set the reactivation key as [systemitem]``ACTIVATION_KEYS``.
+  In the bootstrap script set the reactivation key with the [systemitem]``ACTIVATION_KEYS`` parameter.
   For more information about using reactivation keys, see xref:client-configuration:activation-keys.adoc[].
 
-. OPTIONAL: After the installation of the new proxy, you might also need to:
-+
+After the installation of the new proxy, you might also need to:
+
 * Copy the centrally saved data to the new proxy system
 * Install any other needed software
 * Set up TFTP synchronization if the proxy is used for autoinstallation
 
-[IMPORTANT]
-====
-During the installation of the proxy, clients will not be able to reach the {productname} Server.
-After you have deleted a proxy, the systems list can be temporarily incorrect.
-All clients that were previously connected to the proxy will show as being directly connected to the server instead.
-After the first successful operation on a client, such as execution of a remote command or installation of a package or patch, this information will automatically be corrected.
-This may take some hours.
-====
+
+
+=== Change a Proxy from Traditional to Salt
+
+You can also use re-registration to switch a traditional proxy to a Salt proxy.
```

Comment:
I would use "reinstall" instead of re-registration because I think it could be easy misunderstood with reactivation. I know they are 2 things, but the meaning is so close that is confusing.